### PR TITLE
fix(DB): Bear Flank drop rates

### PR DESCRIPTION
**Pull request by hanabi5**
## Changes Proposed:
- Added missing drop to Rabid Shardtooth
- Changed drop rates according to comunity feedback on tbc classic

## Issues Addressed:

- Closes not reported issue

## SOURCE:

https://tbc.wowhead.com/item=35562/bear-flank#comments

## Tests Performed:

Test not required

## How to Test the Changes:

Killing Bears should grant **Bear Flank** drops to match this table:

image

## Known Issues and TODO List:

None

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by fputs** [comment]
Rabid Shardtooth drops Bear Flank. Killed a bunch and they dropped around 70% of the time, lining up with the table. Still need to test other modified creatures

**Comment by fputs** [comment]
Tested Angerclaw Grizzly and Shardtooth Mauler, both dropped at expected levels. Compared table to SQL changes and they look correct

**Comment by hanabi5** [comment]
Thanks for your help @fputs 👍

File: data/sql/updates/pending_db_world/rev_1650887143894550200.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1650887143894550200');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1650887143894550200.sql:1]: delete this

Comment (hanabi5) [reply data/sql/updates/pending_db_world/rev_1650887143894550200.sql:1]: Ok, but why? don't you use this anymore?

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1650887143894550200.sql:1]: It doesn't exist anymore :)

Comment (hanabi5) [reply data/sql/updates/pending_db_world/rev_1650887143894550200.sql]: Just saying, the pending file comes with it so it should probably be dropped too.

Comment (hanabi5) [reply data/sql/updates/pending_db_world/rev_1650887143894550200.sql]: That was merged 3 hours ago, this was open 4 days ago

**Comment by Nyeriah** [comment]
@hanabi5 please update your branch so we can merge it

**Comment by hanabi5** [comment]
> @hanabi5 please update your branch so we can merge it

Done

**Review by Nefertumm — APPROVED** [review]

**Pull request merged**